types/vhost/vhost-tests.ts

```diff
@@ -1,5 +1,6 @@
+import assert from 'assert';
 import vhost from 'vhost';
 import { Request, Response, NextFunction } from 'express';
 
-vhost('hostname', (_req: Request, _res: Response, _next: NextFunction) => {
-});
+const handler = vhost('hostname', (_req: Request, _res: Response, _next: NextFunction) => {});
+assert(typeof handler === 'function' && handler.length === 3);
```

Comment:
Just to confirm, you know this code is never run, right?  It's only type-checked?  Perhaps you wanted something like `$ExpectType`?

Reply:
No I didn't know. If it is not ran then I can simply remove this last line :)

Reply:
And the import. 😄

Reply:
Ok fixed, thanks!